src/components/server.ts

```diff
@@ -13,6 +13,7 @@ export class Server {
     wsServer: ws.Server
     address: string
     port: number
+    pdfFile: string | undefined = undefined
```

Comment:
What is the purpose of this property? Where do you use it?

Reply:
Unused stuff I forgot to remove after a change of idea...